stdlib/src/builtin/string.mojo

```diff
@@ -572,11 +572,11 @@ fn _is_ascii_lowercase(c: UInt8) -> Bool:
 # ===----------------------------------------------------------------------===#
 
 
-fn isspace(c: UInt8) -> Bool:
+fn _isspace(c: UInt8) -> Bool:
```

Comment:
Can you please mention in the changelog that this is no longer public and `String.isspace` is preferred instead?